cmd/main.go

```diff
@@ -77,6 +78,14 @@ func main() {
 		log.Fatal(err.Error())
 	}
 
+	logLevel, err := logrus.ParseLevel(Options.LogLevel)
+	if err != nil {
+		log.Error("Invalid Log Level: ", Options.LogLevel)
+	} else {
```

Comment:
In this case, I would probably set the log level to default ("info") as if it wasn't provided.

Reply:
Yes nothing will change and the default is info

Reply:
Leaving it as is.